src/applications/caregivers/components/NeedHelpFooter/index.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
```

Comment:
I would consider breaking these out into smaller components as well. For example you could have sections like:

- Caregiver Support
- Caregiver Support Coordinator Location
- National Caregiver Support Line
- Help desk

The general point of this is in keeping with Don't Repeat Yourself. Each of these phone-numbers become hard-coded and non-referenceable. It would be a huge bummer to have to update anything on this page if anything changed. I like to make a CONSTANTS file, probably a json, that contains the one single existence of anything like a phone number or url.